MSAL/src/native_auth/public/MSALNativeAuthPublicClientApplication+Internal.swift

```diff
@@ -119,4 +119,19 @@ extension MSALNativeAuthPublicClientApplication {
         internalChallengeTypes.append(.redirect)
         return internalChallengeTypes
     }
+
+    func copy(with zone: NSZone? = nil) -> Any? {
```

Comment:
@borgesmb can you please clarify why this method is need?

Reply:
it is needed in order to produce a copy of the client application, to be used in the account result. So, the result will have access to the client application, and the get access token method, even in case the user nullify the original client application object.